package.json

```diff
@@ -67,7 +67,7 @@
         "production": "npm start",
         "unlocalize": "grunt unlocalize",
         "test": "grunt test && grunt build-browser-compressed",
-        "start": "http-server -p 8000 --cors"
+        "start": "npm run localize && http-server -p 8000 --cors && npm run unlocalize"
```

Comment:
Note that the only way out of `http-server` is with process termination (ctrl-c), which causes `npm run unlocalize` to never happen. Do we actually need to unlocalize at all?